### PR TITLE
Mpv plugin bookmark 20180320.lua

### PR DESCRIPTION
**Pull request by feijoas**
### mpv-plugin-bookmark
mpv plugin for recording last play in current playing folder and you can resume to play

this plugin will also load the playlist from the playing folder automatically

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by feijoas** [comment]
@jfrankenau Can you review this PR?

**Comment by jfrankenau** [comment]
What is the difference between the upstream light and master branch? The former one is the default one. Also please squash those commits.

File: pkgs/applications/video/mpv/scripts/bookmark.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+stdenv.mkDerivation rec {
+  name = "mpv-plugin-bookmark-${version}.lua";
+  version = "20180320";
+
+  src = fetchFromGitHub {
+    owner = "yozorayuki";
+    repo = "mpv-plugin-bookmark";
+    rev = "08fa0a415a4a220a91b8436ca681e3da92120b9d";
+    sha256 = "0ann6c2rajy36a6w72l2q6288r6v03ha278i5qh5hk6r9q6gkhy8";
```
Comment (Mic92) [inline pkgs/applications/video/mpv/scripts/bookmark.nix:11]: You can return `fetchFromGithub` directly:

Checksum also needs a fix in that case:

```suggestion
    sha256 = "0ann6c2rajy36a6w72l2q6288r6v03ha278i5qh5hk6r9q6gkhy8";
    postFetch = ''
      cp bookmark.lua $out
    '';
    meta = with lib; {
      description = "mpv plugin to record your playing history for each folder and you can choose resume to play next time";
      homepage = https://github.com/yozorayuki/mpv-plugin-bookmark;
      license = licenses.asl20;
      maintainers = [ maintainers.mschneider ];
    };
```

**Comment by Mic92** [comment]
stalled please reopen, if you wish to continue working on this.

**Pull request closed without merging**